plugin/src/test/java/org/wso2/carbon/uuf/maven/bean/AppConfigTest.java

```diff
@@ -60,6 +60,28 @@ public void testLoginPageUriValidations() {
     }
 
     @Test
+    public void testSessionManagerValidations() {
+        AppConfig appConfig = createAppConfig();
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager(""));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("class"));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("c/lass"));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("c*lass"));
+
+        // Null class name
+        appConfig.setSessionManager(null);
+        Assert.assertEquals(appConfig.getSessionManager(), null);
```

Comment:
No point of the assert here as this test case is for validation for session manager configuration.